source/_integrations/tplink.markdown

```diff
@@ -96,13 +101,39 @@ See [Supported Devices in python-kasa](https://github.com/python-kasa/python-kas
 - KL420
 - KL430
 
-Other bulbs may also work, but with limited color temperature range (2700-5000). If you find a bulb isn't reaching the full-color temperature boundaries, submit a bug report to [python-kasa](https://github.com/python-kasa/python-kasa).
+## Supported devices requiring authentication
+
+### Plugs
+
+- EP25 (hw_version >= 2.6)
+- KP125M
+- P110
+- HS100 (UK hw_version = 4.1 fw_version = 1.1.0)
+
+### Wall switches
+
+- KS205
+- KS225
+
+### Bulbs
+
+- L510B
+- L530E
+
+### Light strips
+
+- L900-5
+- L900-10
+
 
 ## Unsupported devices
 
-### Plugs
+Other devices may work but if you encuonter issues submit a bug report to [python-kasa](https://github.com/python-kasa/python-kasa).
+
+Other bulbs may also work, but with limited color temperature range (2700-5000).
+If you find a bulb isn't reaching the full-color temperature boundaries, submit a bug report to [python-kasa](https://github.com/python-kasa/python-kasa).
```

Comment:
I think we could also remove this, as I think the coverage is pretty good, the tapo bulbs deliver this information via the API, and we still log a warning on unknown models.

Reply:
Done